dist/info/beetle_bsnes_libretro.info

```diff
@@ -1,7 +1,7 @@
-display_name = "SNES / Super Famicom (Mednafen bsnes)"
+display_name = "SNES / Super Famicom (Beetle bSNES)"
 authors = "byuu|Ryphecha"
 supported_extensions = "smc|fig|bs|st|sfc"
-corename = "Mednafen/Beetle bsnes"
+corename = "Beetle bSNES"
```

Comment:
While researching emulator histories for Kodi's add-on descriptions, one of the oldest bsnes references I came across stylized it this way. Beetle is a fork of a much more recent version though, so I agree with you that "Beetle bsnes" is fine.

Speaking of which, I'm trying to write brief descriptions for all the cores so we can show stuff in our GUI and also because emulator history fascinates me. For example, two of my descriptions:

Beetle bsnes:
> Beetle bSNES is the SNES module of Beetle, a fork of Mednafen. Mednafen's SNES module itself is a fork of bSNES v0.59.

bSNES-Mercury:
> bSNES Mercury is a fork of Higan (originally called bSNES), notable for emulating the original hardware as accurately as possible through low-level, cycle-accurate emulation.
>
> The "Accuracy" profile focuses on accuracy at the cost of performance.


Would you guys be interested if I shared these descriptions with your .info files?